Microsoft.Toolkit.Uwp.SampleApp/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.1.0",
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
```

Comment:
![screenshot 47](https://cloud.githubusercontent.com/assets/1896163/18254385/195071f2-7395-11e6-886e-bea522ad8cea.png)

before it wouldn't compile.. now it compiled and gave runtime error..